snap/snapcraft-lts.yaml

```diff
@@ -0,0 +1,35 @@
+name: julia
```

Comment:
Doesn't have to be part of this PR but I wonder if we could avoid duplicating the two YAML files by having like a template file that gets edited by CI to contain the version we want.

Reply:
That's a good idea! I would have to think how to better handle this at the CI level, but as you said, that can be done later.

Reply:
We could have `snap/snapcraft-template.yaml` with the version written as something like `${JULIA_VERSION}`, then have a CI step that `sed`s that to the desired version and writes the result to `snap/snapcraft.yaml` (probably gitignored), where the version is determined by parsing `versions.json`. That avoids needing to edit the file for each new patch release as well. Then the only number we'd have to hard-code would be 1.6 for LTS since `versions.json` doesn't flag LTS versions.